src/observer/observers.rs

```diff
@@ -1662,3 +1670,106 @@ pub static CIE2015_10: ObserverData = ObserverData::new(
         [3.407653E-05, 1.337946E-05, 0.000000E+00],
     ])),
 );
+
+#[cfg(test)]
+mod tests {
+    use crate::math::LineAB;
+    use crate::spectrum::SPECTRUM_WAVELENGTH_RANGE;
+
+    use super::*;
+
+    #[test]
+    fn cie1931() {
+        test_observer_spectral_locus_wavelength_range(&CIE1931);
```

Comment:
Ideally I'd want to make it so tests are created "automatically" when you define a new observer. But for now I opted for the more manual approach instead of complicating things.

I did not want to rely on `Observer::iter` to automatically enumerate all `ObserverData` instances. Because I don't want to depend on this other type. I think there is value in being able to verify the correctness of an `ObserverData` without the tag.